src/App.tsx

```diff
@@ -1,10 +1,12 @@
 import { Outlet } from 'react-router-dom'
 import './App.scss'
+import HomeHeader from './components/HomeHeader';
 
 function App() {
   return (
     <div className="homepage">
       <header>Sparkeats Header</header>
+      <HomeHeader />
```

Comment:
FYI, this area is for the Site Header, the Home Header goes [here](https://github.com/sparkbox/sparkeats/blob/main/src/components/home/HomePage.tsx#L7) @arnest00.